vendors/espressif/boards/esp32/CMakeLists.txt

```diff
@@ -349,6 +378,7 @@ target_link_libraries(
         AFR::pkcs11
         AFR::ota_mqtt
         AFR::ota_http
+        AFR::common_io
```

Comment:
Does OTA need common_io?

Reply:
OTA does not need common-io and this change is added by mistake. Actually this change was added when @lundinc2 created a PR to enable common-io on esp32 in my personal fork: https://github.com/shubhamkulkarni97/amazon-freertos/pull/1/files

Sorry for overlooking this change.